Cargo.toml

```diff
@@ -1,18 +1,35 @@
+[workspace]
+members = ["crates/algorithms/*", "crates/web"]
+
+[workspace.package]
+edition = "2024"
+version = "1.1.0"
+authors = ["Brian Anderson <banderson@mozilla.com>", "Andrew Gauger <andygauge@gmail.com>"]
+license = "MIT OR Apache-2.0"
+publish = false
+
 [package]
 name = "rust-cookbook"
 version = "1.1.0"
 authors = ["Brian Anderson <banderson@mozilla.com>", "Andrew Gauger <andygauge@gmail.com>"]
 edition = "2018"
-license = "MIT/Apache-2.0"
+license = "MIT OR Apache-2.0"
 publish = false
 build = "build.rs"
 
 [features]
 default = []
-test-rand = []
 
+[workspace.dependencies]
+rand = "0.9"
+rand_distr = "0.5"
+
+# NOTE: These dependencies add dependencies to the rust playground (play.rust-lang.org).
+# Be wary removing or changing these without considering this fact.
+# See https://github.com/rust-lang/rust-playground/blob/f8d7de52a3c139a0df4fe116bbbff19c30668c99/top-crates/src/lib.rs#L134-L161
```

Comment:
Added this as it's tempting to remove the deps from here as they are not needed (rand / rand-distr are no longer included in the generated skeptic tests).